Modify migration documents to download migration artifacts (#6125)
wso2/docs-apim: en/docs/install-and-setup/upgrading-wso2-api-manager/400-to-410/upgrading-from-400-to-410.md

1. Make sure you backed up all the databases in API-M 4.0.0

2. Download and extract the [migration-resources.zip]({{base_path}}/assets/attachments/install-and-setup/migration/migration-resources.zip). Copy the extracted migration-resources to the `<API-M_4.1.0_HOME>` folder.

3. Copy the [org.wso2.carbon.apimgt.migrate.client-4.1.0.x.jar]({{base_path}}/assets/attachments/install-and-setup/migration/org.wso2.carbon.apimgt.migrate.client-4.1.0.31.jar) file into the `<API-M_4.1.0_HOME>/repository/components/dropins` directory.

    !!! note "If you have configured WSO2 IS 5.11.0 as Resident Key Manager"
        Make sure you have already started the WSO2 Identity Server instance before executing the next step.

> 4. Prior to API-M migration, run the below commad to execute pre migration step which will validate your old data.

Reviewer: ```suggestion
4. Prior to API-M migration, run the below command to execute the pre-migration step that will validate your old data.
```

    api-manager.bat -Dmigrate -DmigrateFromVersion='4.0.0' -DmigratedVersion='4.1.0' 
    -DrunPreMigration
    ```

>     !!! note "If you want to save the invalid api definitions"

Reviewer: ```suggestion
    !!! note "If you want to save the invalid API definitions"
```

>         You can save the invalid api definitions to the local file system during this data validation step if required. Use the `-DsaveInvalidDefinition` option for this as follows. The invalid definitions will be stored under a folder named `<API-M_4.1.0_HOME>/invalid-swagger-definitions` in the form of `<API_UUID>.json`. Then you can manually correct these definitions.

Reviewer: ```suggestion
        You can save the invalid API definitions to the local file system during this data validation step if required. Use the `-DsaveInvalidDefinition` option for this as follows. The invalid definitions will be stored under a folder named `<API-M_4.1.0_HOME>/invalid-swagger-definitions` in the form of `<API_UUID>.json`. Then you can manually correct these definitions.
```
